openmc/mgxs/mgxs.py

```diff
@@ -267,6 +282,19 @@ def __deepcopy__(self, memo):
     def _add_angle_filters(self, filters):
         """Add the azimuthal and polar bins to the MGXS filters if needed.
         Filters will be provided as a ragged 2D list of openmc.Filter objects.
+
+        Parameters
+        ----------
+        filters : Iterable of Iterable of openmc.Filter
+            Ragged 2D list of openmc.Filter objects for which the angle filters
```

Comment:
Reword to something like "Ragged 2D list of Filter objects for the energy and and spatial domains. The angle filters will be added to this list."